chromadb/db/pg.py

```diff
@@ -0,0 +1,544 @@
+from abc import ABC, abstractmethod
```

Comment:
The manual query building seems too verbose IMO, something like https://github.com/kayak/pypika could be a good option, but didn't want to take on another dependency before consulting with the team